tests/unit/mock_data/doom_workflow_slim_metadata.json

```diff
@@ -0,0 +1,2503 @@
+{
```

Comment:
Is there a reason we need a file this big?

Reply:
No reason for its need, this was just a real doomed workflow metadata that was available